src/js/menu/media-chrome-menu.ts

```diff
@@ -103,6 +103,12 @@ template.innerHTML = /*html*/ `
       position: relative;
       bottom: var(--_menu-bottom);
       box-sizing: border-box;
+    } 
+
+    @-moz-document url-prefix() {
+      :host{
+        background: rgb(20 20 30);
+      }
```

Comment:
we still want to honor the CSS vars if they are set `var(--media-menu-background, var(--media-control-background, var(--media-secondary-color` I think. 

so maybe something like
https://codepen.io/luwes-the-looper/pen/NPWNmbY?editors=1100

```css
div {  
  --_menu-bg-color: rgb(20 20 30 / .8);
  
  background: var(--media-menu-background, var(--media-control-background, var(--media-secondary-color, var(--_bg-color))));
}

@-moz-document url-prefix() {
  div {
    --_menu-bg-color: rgb(20 20 30);
  }
}
```